.github/templates/tilgangsinfo-pnd.md

```diff
@@ -7,24 +7,28 @@ labels: 'kind/deploy_patch, team/tilgangsinfo'
 - [ ] Deploy[^1] all components to production[^2].
   - [ ] Deploy [Authentication] to production.
   - [ ] Deploy [Resource Registry] to production.
+  - [ ] Deploy [Auditlog](https://github.com/Altinn/altinn-auth-audit-log/actions) to production.
```

Comment:
Please update this link to use the same format as the other repositories and link directly to the correct action (not just the overview), and add a footnote about how one deploys auditlog to production.